src/utils/hooks.ts

```diff
@@ -63,3 +64,25 @@ export const useMount = (fn: () => void) => {
     //eslint-disable-next-line
   }, []);
 };
+
+export const useHentMalInformasjon = (path: string) => {
```

Comment:
En ulempe med denne funksjonen er at alle vedlegg lastes ned til brukerens maskin for å finne ut av størrelse/filtype... Usikker på om dét egentlig er en god idé...

Filtype kan utledes basert på "filnavnet", men størrelse er naturligvis vanskeligere å utlede uten å laste ned...

Reply:
Har endret til: `fetch(url, { method: 'HEAD' }).then((res) => {` da hentes informasjon fra headers.